deposit-app/src/main/java/edu/unc/lib/boxc/deposit/validate/ExtractTechnicalMetadataJob.java

```diff
@@ -144,6 +147,16 @@ private String getProvidedMimetype(PID originalPid, Model model) {
         }
     }
 
+    private String getProvidedLabel(PID filePid, Model model) {
+        Resource fileResc = model.getResource(filePid.getRepositoryPath());
+        Statement labelStmt = fileResc.getProperty(CdrDeposit.label);
+        if (labelStmt != null) {
+            return labelStmt.getString();
+        } else {
```

Comment:
the `return null;` is needed if labelStmt is null. It could either be in an else or not, I opted for an else clause since its not really an early return, it's one return or the other depending on if labelStmt is present.